src/pages/MapPage.js

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { MapCenterContext } from '../context/MapCenterContext';
+import { Map } from '../components/Map';
+import { LocationsContext } from '../context/LocationsContext';
+import Spinner from '../components/Spinner';
+import ErrorMessage from '../components/ErrorMessage';
+
+const MapPage = () => {
+  const valueCenterMap = useContext(MapCenterContext);
+  const valueLocations = useContext(LocationsContext);
+  const { defaultCenterMap, newCenterMap } = valueCenterMap;
+  const { locations, status, error } = valueLocations;
+
+  return (
+    <>
+      <h1>What's Near Me?</h1>
+      <div className="view__content">
+        {status === 'loading' && <Spinner />}
+        {status === 'error' && <ErrorMessage error={error} />}
+        {(status === 'success' || status === 'cancelled') && (
+          <Map
```

Comment:
I’m not sure I understand why the ``Map`` component code is now both in the ``Map.js`` and ``MapPage.js`` files?

Reply:
@katia-utochkina : We created two files: because `Map.js` as isolate component is easier to use, pass parameters, render our pins, and update the map center point. And `MapPage.js` we needed to create because we had duplicate names.